go-app-ussd_higherhealth_healthcheck.js

```diff
@@ -161,26 +162,46 @@ go.app = (function () {
     self.states.add("state_timed_out", function (name, creator_opts) {
       return new MenuState(name, {
         question: $([
-          "Welcome back to HealthCheck",
+          "Welcome back to the The National Department of Health's COVID-19 Service",
           "",
           "Reply"
         ].join("\n")),
         accept_labels: true,
         choices: [
           new Choice(creator_opts.name, $("Continue where I left off")),
-          new Choice("state_start", $("Start over"))
+          new Choice("state_language", $("Start over"))
         ]
       });
     });
 
+    self.add("state_language", function(name){
+        return new LanguageChoice(name, {
+        question: $([
+          "Welcome to HealthCheck.",
+          "Please select your preferred language.",
+
```

Comment:
You need a `"",` on this line, otherwise you won't end up with the extra new line that you want here.